content/updates/productivity/1-An-even-more-powerful-product-grid.md

```diff
@@ -0,0 +1,52 @@
+# An even more powerful product grid
+
+## Duplicate your products
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+To easily create a new product based on an existing one, you can now use our duplicate feature to copy a product in your user interface in one single click!
+
+Find this new action under the `...` button in the top right corner of each of your product edit forms.
+
+![Duplicate action in the `...` menu](../img/duplicate-action.png)
+
+This new feature also comes with dedicated permission, so you can decide which group(s) of users can perform this action.
+
+## Create private views
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/manage-your-views.html#create-a-view"
+
+Stay organized in your product enrichment thanks to private views! From now on, when you create a view, you can make it private to keep it for yourself and have it only visible by you, or public to share it with other PIM users. This improvement will help you to focus on your own enrichment needs without confusing your teammates. 
+
+![Create a private view](../img/private-view.png)
+
+## Switch between simple and variant product displays
```

Comment:
I would put this section just after the next one, as you are referring to the "grouped" display option.

Reply:
Et peut-être qu'on pourrait changer un peu le titre pour bien le différencier avec celui d'après. Je ne sais pas comment le formuler en anglais...

Reply:
@camillebourjade any ideas?